tests/zmq/test_gnmi_zmq.py

```diff
@@ -9,6 +9,7 @@
 
 
 pytestmark = [
+    pytest.mark.disable_loganalyzer,
```

Comment:
Arista has fixed this error log in newest release.

Do we have clue on why this test could trigger errors on other platforms?
We usually prefer to fix the error log or suppress the specific error log pattern in log analyzer instead of disabling it.
But it would make sense to disable it if the test itself could trigger error logs by nature (such as restart some critical services).

Reply:
This test will reload config during test, which will restart all service. I think this is the reason why the error been triggered.
There are 2 kinds of error break this test, so if we don't disable log analyzer there may similar issue break this test again.